### PR TITLE
fix: Improve link preview image display

### PR DESCRIPTION
**Pull request by atomrc**
### Before
image

### After
image

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### CODE REVIEW

File: src/style/components/asset/link-preview-asset.less [hunk 1]
```diff
@@ -32,7 +32,8 @@
 }
 
 .link-preview-image-container {
-  width: @link-preview-height;
+  min-width: @link-preview-height;
+  height: @link-preview-height;
```
Comment (V-Gira) [inline src/style/components/asset/link-preview-asset.less:36]: let's change the variable to rem, the reason height wasn't fixed was for larger font sizes.

80px would be 5rem, wdyt?

Comment (atomrc) [reply src/style/components/asset/link-preview-asset.less:36]: Oh that's why we had this strange : `width: @...-height` :D

Comment (atomrc) [reply src/style/components/asset/link-preview-asset.less:36]: Using rem sounds like a nice solution 👍

**Review by V-Gira — APPROVED** [review]
Let's see how that looks :)

File: src/style/components/asset/link-preview-asset.less [hunk 2]
```diff
@@ -18,7 +18,7 @@
  */
 
 // variables
-@link-preview-height: 88px;
+@link-preview-height: 5.5rem;
```
Comment (V-Gira) [inline src/style/components/asset/link-preview-asset.less:21]: Oh, I read 80px for some reason, good on ya for double checking :D

**Review by PatrykBuniX — APPROVED** [review]

**Review by V-Gira — APPROVED** [review]

**Pull request merged**